bin/cliain/src/commands.rs

```diff
@@ -321,6 +328,9 @@ pub enum Command {
     /// API signature: https://polkadot.js.org/docs/substrate/extrinsics/#calldest-multiaddress-value-compactu128-gas_limit-compactu64-storage_deposit_limit-optioncompactu128-data-bytes
     ContractCall(ContractCall),
 
+    /// Returns OwnerInfo if code hash is stored on chain
+    ContractOwnerInfo(ContractOwnerInfo),
```

Comment:
you can also add 
```rust
        | Command::ContractOwnerInfo { .. }
```
to the pattern-match in `read_seed`, since you don't need any signer for the connection (now it requires some seed)

Reply:
done!